posts/2020/03/uarray-gsoc-participation.md

```diff
@@ -0,0 +1,16 @@
+<!--
+.. title: uarray: GSoC Participation
+.. slug: uarray-gsoc-participation
+.. date: 2020-03-12 13:39:00 UTC+01:00
+.. tags: uarray, SciPy, NumPy, Tensors, Big data
+.. type: text
+.. author: Hameer Abbasi
+-->
+
+I'm pleased to announce that `uarray` is participating in GSoC '20 as a sub-organization under the umbrella of the Python Software Foundation. Our ideas page is up [here](https://uarray.org/en/latest/gsoc/2020/ideas.html), go take a look and see if you (or someone you know) is interested in participating, either as a student or as a mentor.
+
+Myself, Prasun Anand and Peter Bell will be mentoring, and we plan to take a maximum of two students, unless more community mentors show up.
+
+There have been [quite](https://github.com/Quansight-Labs/uarray/pulls?q=is%3Apr+is%3Aclosed) [a few](https://github.com/Quansight-Labs/unumpy/pulls?q=is%3Apr+is%3Aclosed) [pull requests](https://github.com/Quansight-Labs/udiff/pulls?q=is%3Apr+is%3Aclosed) already to qualify from prospective students, some even going as far as to begin the work described in the idea they plan to work on. However, at the time of writing, no students had submitted an actual proposal.
```

Comment:
> However, at the time of writing, no students had submitted an actual proposal.

This is normal, so I would perhaps leave that sentence out. As well as the "Even so" in the next paragraph.